modules/end-to-end-test/src/test/groovy/org/hawkular/integration/test/Scenario1ITest.groovy

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.hawkular.integration.test
+
+import java.util.Map;
+
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.impl.client.LaxRedirectStrategy;
+import org.hawkular.inventory.api.model.Environment
+import org.hawkular.inventory.api.model.Metric
+import org.hawkular.inventory.api.model.MetricType
+import org.hawkular.inventory.api.model.MetricUnit;
+import org.hawkular.inventory.api.model.Resource
+import org.hawkular.inventory.api.model.ResourceType
+import org.hawkular.inventory.api.model.Tenant
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Assert;
+import org.junit.Test
+
+
+class Scenario1ITest extends AbstractTestBase {
+
+    static final String urlTypeId = "URL"
+    static final String environmentId = "test"
+    static final String statusCodeTypeId = "status.code.type"
+    static final String durationTypeId = "status.duration.type"
+
+    @Test
+    public void testScenario() throws Exception {
+        //def response = client.get(path: "/hawkular-accounts/organizations")
+        def response = client.get(path: "/hawkular-accounts/personas/current")
+        assertResponseOk(response.status)
+        String tenantId = response.data.id
+        // println "tenantId = $tenantId"
+
+        /* assert the test environment exists */
+        response = client.get(path: "/hawkular/inventory/$tenantId/environments/$environmentId")
+        assertResponseOk(response.status)
+        org.junit.Assert.assertEquals(environmentId, response.data.id)
```

Comment:
Ehm... because editing Groovy kinda sucks. But yes, for the sake of the pure beauty, I can import them :)